custom/aaa/tests/test_utils.py

```diff
@@ -0,0 +1,202 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from django.test.testcases import TestCase
+
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.locations.models import LocationType, SQLLocation
+from custom.aaa.const import MINISTRY_MWCD, ALL, MINISTRY_MOHFW
+from custom.aaa.utils import build_location_filters
+
+
+class TestUtils(TestCase):
```

Comment:
Should also be a national test for both hierarchies where no locaiton filter is given